package.json

```diff
@@ -58,7 +58,8 @@
   "peerDependencies": {
     "@material-ui/core": "^4.10.2",
     "react": "^16.9.0",
-    "react-dom": "^16.9.0"
+    "react-dom": "^16.9.0",
+    "react-router-dom": "^5.2.0"
```

Comment:
You also need it as a main dependency, not just peer, you'd need to have it both under `dependencies` and `peerDependencies`

Reply:
Keep the version same